homework/shelkovich_ilya/userValidationService/src/main/java/com/tutrit/java/quickstart/bean/User.java

```diff
@@ -0,0 +1,63 @@
+package com.tutrit.java.quickstart.bean;
+
+import java.util.Objects;
+
+public class User {
+    String firstName;
+    String lastName;
+    int age;
```

Comment:
Make fields as private. Best practicks use the incapsulate.
https://refactoring.guru/ru/encapsulate-field
https://metanit.com/java/tutorial/3.3.php

Reply:
from now as we learned 4 principles OOP and scopes, all times fields should be private, unless you know for sure why it shouldn't. Please, modify.

Reply:
fixed it